src/main/java/seedu/address/ui/GroupListPanel.java

```diff
@@ -25,6 +27,14 @@
      */
     public GroupListPanel(ObservableMap<Name, Group> groupMap) {
         super(FXML);
+        //Prevent mouse selection
```

Comment:
nitpick: can we standardise whether to leave a new line here and keep it the same throughout the UI components?

Reply:
Agree with ivan that we should add a space are the `super(FXML)` line for consistency